doc/developer_guide/developer_guide.rst

```diff
@@ -1,2 +1,79 @@
-Developer Guide
-===============
+.. _developer_guide:
+
+:octicon:`tools` Developer Guide
+================================
+
+Creating a Release
+++++++++++++++++++
+
+Prepare the Release
+-------------------
+
+To prepare for a release, a pull request with the following parameters needs to be created:
+
+- Updated version numbers
+- Updated the changelog
+- Updated workflow templates (not automated yet)
+
+This can be achieved by running the following command:
+
+.. code-block:: shell
+
+   nox -s prepare-release -- <major>.<minor>.<patch>
```

Comment:
I would hope to have found this in the PTB and just put a link here :slightly_frowning_face: 
For now it's better to have it here. 
Thanks!

Reply:
Maybe also mention the CLI options `--no-branch` and `--no-pr`?

Reply:
I think it is much more convenient to have the entire workflow for the release included in the project documentation, even though this means some duplication.